examples/tv-casting-app/linux/main.cpp

```diff
@@ -133,6 +158,9 @@ void PrepareForCommissioning(const Dnssd::DiscoveredNodeData * selectedCommissio
     // Display onboarding payload
     chip::DeviceLayer::ConfigurationMgr().LogDeviceConfig();
 
+    // Initialize binding handlers
+    InitBindingHandlers();
```

Comment:
Are these changes related to ACL?

Reply:
these relate to bindings. Updated title of PR